WcaOnRails/app/views/search_results/index.html.erb

```diff
@@ -77,6 +77,21 @@
       <%= paginate @posts, param_name: :posts_page, params: { anchor: "posts" } %>
     <% end %>
 
+    <% if !@regulations.empty? %>
+      <h2><%= anchorable "Regulations and Guidelines" %></h2>
+      <% @regulations.each do |reg| %>
+        <div>
+          <%= link_to reg["url"] do %>
+            <h4><%= wca_highlight(reg["id"], query_parts) %></h4>
+          <% end %>
+          <blockquote>
+            <%= wca_highlight(reg["text"], query_parts) %>
```

Comment:
I had to tweak it a bit because it would fail if the string doesn't contain html, see [here](https://github.com/cubing/worldcubeassociation.org/pull/722/commits/3366007f17f1557279634f70126d354c1ea10dce#diff-899eee8f20888bf9629deb12fd5852e9R81) for the solution ;)

If it looks good to you I'll merge this up!